build.gradle

```diff
@@ -4040,6 +4044,27 @@ project(":systemTests") {
     addValidateSourceSets(project, nonModSrcSets, modSrcSets)
 }
 
+project(":3DLighting") {
+
+    apply plugin: "application"
```

Comment:
I hadn't meant to suggest adding it to the main build.gradle. I think this is not the best place for it. I was thinking of something along the lines of a build.gradle or build.xml in the`tests/performance/3DLighting` dir which is what the other manual tests do -- at least those that are more complicated than just a collection of files in a single directory.